brd-ios/breadwallet/src/Helpers/SimpleTextField.swift

```diff
@@ -140,6 +176,19 @@ class SimpleTextField: UIView, UITextFieldDelegate {
         textField.layer.borderColor = isVisible ? UIColor.kycGreen.cgColor : UIColor.kycGray1.cgColor
     }
     
+    func setEmptyErrorMessage(isFieldEmpty: Bool) {
+        errorLabel.isHidden = !isFieldEmpty
+        if isFieldEmpty {
+            textField.layer.borderColor = UIColor.red.cgColor
+        }
```

Comment:
there is a similar login above

`textField.layer.borderColor = isVisible ? UIColor.kycGreen.cgColor : UIColor.kycGray1.cgColor`

you can do it like that with red and clear color. 
please add the red color to constants.